src/main/scala/org/downtowndailybread/bethsaida/request/util/DatabaseRequest.scala

```diff
@@ -14,6 +18,30 @@ trait DatabaseRequest {
     }.toSeq
   }
 
+  class EnhancedPreparedStatement(ps: PreparedStatement) {
+    def setNullableInt(parameterIndex: Int, x: Option[Int]): Unit = {
+      x match {
+        case Some(i) => ps.setInt(parameterIndex, i)
+        case None => ps.setNull(parameterIndex, java.sql.Types.INTEGER)
+      }
+    }
+    def setNullableString(parameterIndex: Int, x: Option[String]): Unit = {
+      x match {
+        case Some(i) => ps.setString(parameterIndex, i)
+        case None => ps.setNull(parameterIndex, java.sql.Types.VARCHAR)
+      }
+    }
+    def setNullableUUID(parameterIndex: Int, x: Option[UUID]): Unit = {
+      x match {
+        case Some(i) => ps.setString(parameterIndex, i)
```

Comment:
Does this automatically convert the UUID to a string?

Reply:
ya, UUIDProvider on line 11:

```
implicit def uuidToString(uuid: UUID): String = uuid.toString
```